jwst/resample/resample_spec.py

```diff
@@ -390,7 +390,9 @@ def do_drizzle(self, xmin=0, xmax=0, ymin=0, ymax=0, **pars):
             output_model.meta.resample.weight_type = self.drizpars['weight_type']
             output_model.meta.resample.pointings = pointings
 
-            # Update mutlislit slit info on the output_model
+            # Update mutlislit slit info on the output_model. This is needed
```

Comment:
Not your fault, but there's a typo in `mutlislit`

Reply:
Fixed.